internal/testsuite/otaa_test.go

```diff
@@ -781,7 +781,7 @@ func (ts *OTAATestSuite) TestLW11() {
 				conf := test.GetConfig()
 				conf.JoinServer.KEK.Set = []struct {
 					Label string
-					KEK   string `mapstructure:"kek"`
```

Comment:
Was there a reason to remove this? I believe this is causing the tests to fail:

```
vet: internal/testsuite/rejoin_test.go:87:28: cannot use ([]struct{Label string; KEK string} literal) (value of type []struct{Label string; KEK string}) as []struct{Label string "mapstructure:\"label\""; KEK string "mapstructure:\"kek\""} value in assignment
```

Reply:
@brocaar thanks for your comments. There was no reason, it was my foolishness when creating the commit.